shell/browser/api/atom_api_session.cc

```diff
@@ -763,7 +803,8 @@ void Session::BuildPrototype(v8::Isolate* isolate,
       .SetProperty("cookies", &Session::Cookies)
       .SetProperty("netLog", &Session::NetLog)
       .SetProperty("protocol", &Session::Protocol)
-      .SetProperty("webRequest", &Session::WebRequest);
+      .SetProperty("webRequest", &Session::WebRequest)
+      .SetMethod("preconnect", &Session::Preconnect);
```

Comment:
Please put this with the other methods above.

Reply:
ok